app/ldap.js

```diff
@@ -564,6 +564,11 @@ exports.addGroup = function (group, cb) {
   });
 };
 
+// check if LDAP is disabled
+exports.isDisabled = function() {
+  return (!process.env.NODE_ENV || 'development' === process.env.NODE_ENV) && 'true' !== process.env.LDAP;
```

Comment:
My reasoning for axing the first conditional is that it always is true -- why check it? 

I edited my comment (see right above yours) -- put back in the parenthesis. -- you **DO NOT** need to check for the presence of NODE_ENV -- it's always there.